README.md

```diff
@@ -7,16 +7,26 @@ The Ansible FRR collection includes a variety of Ansible content to help automat
 
 This collection has been tested against FRR 6.0.
 
-### Supported connections
+<!--start requires_ansible-->
+## Ansible version compatibility
+
+Plugins and modules within a collection may be tested with only specific Ansible versions. 
+A collection may contain metadata that identifies these versions. 
+PEP440 is the schema used to describe the versions of Ansible.
+
```

Comment:
I'm wondering how much of the above boilerplate matters to an end user.  For example, if we list a tested version, doesn't that mean all the collection plugins/modules have been tested against those versions?  Is there a case where we will list the tested version, but not all modules have been tested?

Reply:
This was an attempt to try and provide some context around the tested with line and further clarify what tested with means. The PEP440 reference was added in case the meaning of string needed clarification. I had originally tried to parse the string and convert it to words, but that proved to be more work than what it was bringing